plugins/outputs/cloudwatch/aggregator_test.go

```diff
@@ -109,12 +109,12 @@ func TestAggregator_MultipleAggregationPeriods(t *testing.T) {
 	aggregator.AddMetric(m)
 
 	assertNoMetricsInChan(t, metricChan)
-	assertMetricContent(t, metricChan, aggregationInterval * 2, m, expectedFieldContent{"value", 3, 1, 3, 6, "",
+	assertMetricContent(t, metricChan, aggregationInterval * 3, m, expectedFieldContent{"value", 3, 1, 3, 6, "",
```

Comment:
I'm mildly suspicious of this because I saw in https://github.com/aws/amazon-cloudwatch-agent/pull/334/commits/da00fff5c83eb5aa660a654b2236dca7de1561eb that it did not fix the test. I've been rerunning the MacOS workflow on this PR when I remember to and it passes consistently so not sure why it failed more often than succeeded with a larger timeout from the PR that originally refactored this. Part of me thinks it could be more related to the additional sleep time on line 143 where there wasn't a sleep before. 

I would rather figure out the root issue than play around with sleep timers on unit tests, but don't want this to become a cause of significant confusion for people submitting PRs. 

For my own sanity, can you change the 1 second sleep you added back to 0 seconds and see if it still consistently passes in GitHub CI? I am curious if that test has cascading issues that affect these other tests. I haven't dug around the source code this is testing too much.